ports/libsimpleservo/capi/src/lib.rs

```diff
@@ -874,4 +902,16 @@ impl HostTrait for HostCallbacks {
             },
         }
     }
+
+    fn show_context_menu(&self, items: Vec<String>) {
+        debug!("show_context_menu");
+        let size = items.len() as u32;
+        let cstrs: Vec<CString> = items
+            .into_iter()
+            .map(|i| CString::new(i).expect("Can't create string"))
+            .collect();
+        let ptrs: Vec<*const c_char> = cstrs.iter().map(|cstr| cstr.as_ptr()).collect();
+        (self.0.show_context_menu)(ptrs.as_ptr(), size);
+        let _ = cstrs; // Don't drop these too early
```

Comment:
This line doesn't look like it should actually be necessary - won't cstrs last until the end of function by default?

Reply:
Hmm… I tried again, and indeed `cstrs` is not dropped early.
For some reason this was necessary at some point, not sure why.

I removed the line.